dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.websphere.appserver.jakarta-connectionManagement-1.0.feature

```diff
@@ -6,5 +6,5 @@ IBM-Provision-Capability:\
 IBM-Install-Policy: when-satisfied
 -features=com.ibm.websphere.appserver.jakarta.connector-2.0
 -bundles=com.ibm.ws.jca.cm.jakarta
-kind=noship
+kind=beta
 edition=full
```

Comment:
this should be moved to `edition=base`

Reply:
same comment applies to all other feature files too (jdbc features should go into `core` and jca features should go into `base)

Reply:
As long as the spec features upon which they depend remain edition=full, these need to remain edition=full.  It might be correct to change those as well, but I'm not going to do it under this pull.  That needs to be a separate discussion.